components/AirMapRadio.js

```diff
@@ -61,6 +55,9 @@ const styles = StyleSheet.create({
     alignItems: "center",
     justifyContent: "center",
   },
+  radiobutton: {
+    paddingRight: 25,
```

Comment:
Great work! Please try to stick to percentages for styles. it will be important when working with different view points.